source/jsonnet/northern-ireland/individual/blocks/employment/place_of_work.jsonnet

```diff
@@ -0,0 +1,93 @@
+local placeholders = import '../../../lib/placeholders.libsonnet';
+local rules = import 'rules.libsonnet';
+
+local ireland = 'No, it is in the Republic of Ireland';
+local pastIreland = 'No, it was in the Republic of Ireland';
+
+local anotherCountry = 'No, it is in another country';
+local pastAnotherCountry = 'No, it was in another country';
```

Comment:
The variable should be named such that it is clear what it is for.

Reply:
I named them differently so they are more descriptive now, hopefully.